frontend/src/declarations.d.ts

```diff
@@ -34,15 +34,6 @@ declare module "@loaders.gl/csv"
 
 declare module "@loaders.gl/gltf"
 
-declare module "@emotion/styled" {
-  import { CreateStyled } from "@emotion/styled/types/index"
-  import { Theme } from "src/theme"
-
-  export * from "@emotion/styled/types/index"
-  const customStyled: CreateStyled<Theme>
-  export default customStyled
-}
-
```

Comment:
curious why this is no longer needed?

Reply:
Ah yes. This is removed in replacement for the emotion.d.ts file (explanation for why it was moved is in that file).